.github/workflows/manual-deploy.yml

```diff
@@ -0,0 +1,39 @@
+name: Deploy to AKS (Manual)
+concurrency: manul_deploy_${{ github.event.inputs.environment }}
+
+on:
+  workflow_dispatch:
+    inputs:
+      docker-image-tag:
+        description: 'Docker image tag'
```

Comment:
Please remove quotes

Reply:
Should be addressed with [7cb75b1](https://github.com/DFE-Digital/teacher-pay-calculator/pull/18/commits/7cb75b1c5ce7bdd8f6cb0f25ed40b50c88803370) now